package.json

```diff
@@ -1,8 +1,10 @@
 {
+  "name": "monorepo",
   "license": "Apache-2.0",
-  "private": true,
   "workspaces": [
     "packages/*"
   ],
-  "packageManager": "yarn@3.1.0"
+  "scripts": {
+    "test": "yarn workspaces foreach run test"
```

Comment:
@vojtechszocs @andrewballantyne if we are planning on using a mono-repo structure, should we investigate using something like [Lerna](https://github.com/lerna/lerna)? It'll make running things like this easier IMO.

Reply:
@florkbr For now, I've used [`yarn workspaces foreach`](https://yarnpkg.com/cli/workspaces/foreach) CLI (provided via Yarn `workspace-tools` plugin) instead of Lerna for the sake of simplicity.

I agree that we should look into tools like Lerna in the future, based on what we need to do with our packages. This monorepo should be quite simple - some packages are libraries (to be published to npm package registry) while others are helpers. It makes sense to have `build` and `test` scripts for all of these packages.